en/contributing-build.md

```diff
@@ -0,0 +1,52 @@
+Title: Build the Docs
+TODO: can add more user-friendly instructions on setting up local web server
+      talk about entr utility
+
+
+# Build the Docs
+
+Every non-trivial contribution must first have its HTML built and verified
+before a pull request (PR) is made from it.
+
+First install the builder. On Ubuntu 16.04 LTS:
+
+```bash
+sudo snap install documentation-builder
+```
+
+To build the HTML, while in the root of the MAAS docs repository:
+
+```bash
+/snap/bin/documentation-builder
+```
+
+See the [documentation-builder project][github-documentation-builder] for
+details.
+
+You will now need a web server. See the
```

Comment:
Do you think it may be worth mentioing that you don't necessarily need to run a web server,  and that you can simply load index.html (or any html file) into your browser directly?